source/_patterns/01-atoms/svgicon/demo/_svgicon-demo.scss

```diff
@@ -0,0 +1,44 @@
+// All component Sass needs non-printing base config
+@import '../../../00-protons/variables';
+@import '../scss/_icons-generated';
+@import '~font-awesome/scss/mixins';
+@import '~font-awesome/scss/variables';
+
+.svgicon-demo-0 {
+  @include sprite('chevron-right');
+  height: 40px;
+  width: 40px;
+}
+
+.svgicon-demo-1 {
+  @include sprite(
```

Comment:
I know that `sprite()` is auto-generated, but that name is sooooo easily stepped-on...